java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/refactoring/MoveRefactoring.java

```diff
@@ -118,6 +120,11 @@
         }
         info.toPhase(JavaSource.Phase.ELEMENTS_RESOLVED);
         int offset = getOffset(info, params.getRange().getStart());
+        TokenSequence<JavaTokenId> ts = info.getTokenHierarchy().tokenSequence(JavaTokenId.language());
```

Comment:
4th occurrence of the same block in this diff in different files - if this adjustment is mandatory / desirable when going from LSP Position to Element, shouldn't we make an utility method ?

Reply:
This adjustment is desirable just in the particular cases - for example having a cursor placed just before a method invocation `  |foo();`, the code completion needs to return result valid for the scope of the surrounding block. On the other hand, features like hover, refactorings, etc. want for the same offset to operate on the method call...